pyfolio/utils.py

```diff
@@ -259,3 +260,45 @@ def extract_rets_pos_txn_from_zipline(backtest):
     transactions.index.tz = None
 
     return returns, positions, transactions, gross_lev
+
+
+SETTINGS = {
+    # Settings dict to store functions/values that may
+    # need to be overridden depending on the users environment
+    'returns_func': default_returns_func
+}
+
+
+def register_return_func(func):
+    """
+    Registers the function that will be called for
+    retrieving returns data.
+
+    Parameters
+    ----------
+    func : function
+        A function that accepts a symbol and
+        returns a pandas.Series of asset returns.
+
+    Returns
+    -------
+    None
+    """
+    SETTINGS['returns_func'] = func
+
+
+def get_symbol_rets(symbol):
```

Comment:
I was initially thinking `*args, **kwargs` would make sense. A use case I can think of is somebody querying an internal database, they might need to pass db/table names or dates. What if we made the signature 
`get_symbol_rets(symbol, args=None, kwargs=None)`
Then the user would be expected to pass a tuple and/or dict of their required parameters. 

I was also planning on adding start/end dates to this, I'm sure data since 1970 isn't always required.